lumigator/python/mzai/backend/backend/api/routes/completions.py

```diff
@@ -1,5 +1,5 @@
 from fastapi import APIRouter
-from schemas.completions import CompletionRequest
+from lumigator_schemas.completions import CompletionRequest
```

Comment:
curious why we decided to rename schemas. sdk renaming makes sense since we're shipping as a top-level package.

Reply:
Well, basically the same reason, and consistency :-/ I'm sure there are lots of packages named `schemas` out there :D If you'd rather go back to plan `schemas`, just let me know.

Currently we don't have a standalone lumigator_schemas package as part of the SDK, btw, but as a _module_ inside the sdk package. This is one thing we can discuss.

Reply:
And another one inside backend too. Should we do something about that now?